### PR TITLE
Added CFBundlePackageType and NSHighResolutionCapable by default to Info.plist of Darwin bundles

### PR DESCRIPTION
**Pull request by cainesi**
This PR adds two items to the Info.plist file of an application Bundle:
1. CFBundlePackageType: APPL, which specifies that the bundle is an application bundle
2. NSHighResolutionCapable: True, which tells the application to run in high-resolution mode by default (which seems like a more reasonable default in 2021--without this frozen GUI apps can appear blurry).

### CODE REVIEW

**Comment by marcelotduarte** [comment]
Does this patch fix #239?

**Comment by cainesi** [comment]
Yes it does!

(I've been fixing it in my own apps via the plist_items parameter for a while. Did not see that there was an open issue for it.)

**Comment by marcelotduarte** [comment]
Good!
I think that the previous title "Added CFBundlePackageType and NSHighResolutionCapable to an Darwin bundles Info.plist." should be better, no?

File: cx_Freeze/macdist.py [hunk 1]
```diff
@@ -237,6 +237,12 @@ def create_plist(self):
         # Ensure CFBundleExecutable is set correctly
         contents["CFBundleExecutable"] = self.bundle_executable
 
+        # Specify that bundle is an application bundle
+        contents["CFBundlePackageType"] = "APPL"
+
+        # Cause application to run in high-resolution mode by default
+        contents["NSHighResolutionCapable"] = "True"
```
Comment (marcelotduarte) [inline cx_Freeze/macdist.py]: Is this value set to true but the user can change it in the plist_items?
Can you document this?

**Comment by cainesi** [comment]
Yes, fine with me.

File: cx_Freeze/macdist.py (hunk 1 above)
Comment (cainesi) [reply cx_Freeze/macdist.py]: Done.  I also tweaked the code so that the new default values will not be used if there is a custom_info_plist specified.

**Comment by marcelotduarte** [comment]
Thanks!

**Pull request merged**